.eslintrc.js

```diff
@@ -17,13 +17,12 @@ module.exports = {
     afterEach: 'readonly'
   },
   root: true,
-  extends: [
-    'eslint:recommended',
-    'plugin:@typescript-eslint/eslint-recommended',
-    'plugin:@typescript-eslint/recommended',
-    'prettier/@typescript-eslint'
-  ],
+  extends: ['eslint:recommended', 'plugin:@typescript-eslint/recommended'],
```

Comment:
Why not keeping prettier rules?

```suggestion
  extends: ['eslint:recommended', 'plugin:@typescript-eslint/recommended', 'prettier'],
```

Reply:
I was just going off the `eslint` recommendations. We still have a prettier command that runs. It might be that I should put this rule back in.